.github/workflows/lint-test.yml

```diff
@@ -97,12 +97,9 @@ jobs:
         --format='::error file=%(path)s,line=%(row)d,col=%(col)d::\
         [flake8] %(code)s: %(text)s'"
 
-      # We run `coverage` using the `python` command so we can suppress
-      # irrelevant warnings in our CI output.
       - name: Run tests and generate coverage report
         run: |
-            python -Wignore -m coverage run -m unittest
-            coverage report -m
+            pytest -n auto --cov --disable-warnings -q
```

Comment:
This doesn't have to be on a separate line any more but whatever.

Reply:
Might as well fix it.